### DISCUSSION TITLE
Filter only places inside specific country / countries

### DISCUSSION THREAD

**Discussion by martinsnajdr** [systemed/tilemaker#680] [Q&A]
Hello,

First of all, I just want to thank you for the Tilemaker. It's really an amazing tool.

I would like to ask if there is a way to export to mbtiles the information in which country the place is located? I looked in OSM inspector to see what data is stored for places (cities, etc) and I couldn't find the country, which I guess is actually logical.

My point is that I'd like to show bits of countries outside of one particular country, but I don't want to show place names outside of my particular country because it would be unnecessarily distracting. I know that you can download specific countries from geofabrics, but that then means that the country borders are cropped from the surrounding area, which again I don't completely want.

I was thinking of filtering by localized names like name:cs and such, but of course some cities abroad have local (in this case Czech) names as well, so there's no certainty that name:cs would only filter out cities within the Czech Republic. 

It would probably somehow plug in the boundaries data from OSM boundaries, but that seems like an unnecessarily complicated method to me. 

(I'm working with the openmaptiles schema within tilemaker)

Thank you!

**Comment by systemed** [comment]
Yes - you can do this by supplying a shapefile or GeoJSON file with your country boundary, and then using tilemaker's spatial query support. See here: https://github.com/systemed/tilemaker/blob/master/docs/CONFIGURATION.md#shapefiles-and-geojson
